eth/api_debug.go

```diff
@@ -89,24 +89,42 @@ func (api *DebugAPI) DumpBlock(blockNr rpc.BlockNumber) (state.Dump, error) {
 	return stateDb.RawDump(opts), nil
 }
 
-func (api *DebugAPI) ExecutionWitness(blockNr rpc.BlockNumber) (*stateless.Witness, error) {
-	witness := &stateless.Witness{}
-
-	block := api.eth.blockchain.GetBlockByNumber(uint64(blockNr))
+func (api *DebugAPI) ExecutionWitness(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) (hexutil.Bytes, error) {
```

Comment:
`hexutil.Bytes` hex is 2x larger than raw bytes... consider using Base64, or some other format?

Reply:
As per our talk with reth & op-geth folks, we agreed on to directly return the json data for simplicify (performance currently is not a concern here).

We can implement the deserializer